Adapter/Common.php

```diff
@@ -95,6 +95,10 @@ public function cropResize($width = null, $height = null, $background = 'transpa
      */
     public function fixOrientation()
     {
+        if (!in_array(exif_imagetype($this->source->getInfos()), array(IMAGETYPE_JPEG, IMAGETYPE_TIFF_II, IMAGETYPE_TIFF_MM))) {
+            return $this;
```

Comment:
It would be better to throw an unsupported image type exception IMO.

@Gregwar what do you think?